io/swig/io/vview.py

```diff
@@ -918,22 +890,36 @@ def init_cf_sources(mu):
         transforms[instance] = []
         offsets[instance] = []
         actors[instance] = []
```

Comment:
Just need to remove this line and then it works for me. ("NameError: name 'actors' is not defined")